### ISSUE TITLE
Adding Spikes To Police Room

### ISSUE THREAD

**Issue by Terston** [DDNetPP/maps#14]
Police room isn't fun because it barely holds any value.
The police room barely holds any value because its very squishy and too easy to retake control of.
The reasons its too easy to retake are:
- Cannot WB (Should stay this way)
- There are no kill tiles

How does having no kill tiles ruin the purpose of the Police Room?
- Because there are no kill tiles, and you cannot way block, all it takes is 1 person to come and help all the freezed people really quick, even if 5 experienced players take control of the police room, its too easy to help everyone and create total chaos.
I play this server for quite a while now, and no-one is trying to raid the police room. Why? - Because its too easy getting there.

Why should the Police Room hold more value?
- Because in the Police Room you can get more money, and more xp, hence increasing the value of the room. If its that easy to retake the police room, and almost impossible to control it, afk people cannot farm for money/xp safely.

More evidence:

- The reason BlmapV5 was so popular back in the day, is because of arena 5. In arena 5 you could wb, and it was almost impossible to retake. It was- 'precious'. It had value to it. There was a reason to control this place, without even caring for farming. It was the room for the pro people in the server. If the Police Room doesn't hold the same value, why do we get bonus money and xp there? It makes no sense.
If you add value to the police room, people will actually care to control it, and it wont even be about farming (which is also quite impossible to do for long periods as it takes 5 minutes for an experienced player to take control of the whole Police Room), take the players 'HeavyGeneral', 'Shiro.' as an example.

- Me (Terston) and HeavyGeneral, raided the police a lot of times, and we can block people quite easily, especially now that tasers are nerfed, even v10 people- we can handle it, but its enough for 1 person to come and help all these 10 people, making it all pointless.

An idea for where to put kill tiles:
![156_LI](https://user-images.githubusercontent.com/24831610/97745302-588ce000-1af1-11eb-85be-c417a94c7883.jpg)

An alternative would be to make helping blocked tees harder in police.

**Comment by ChillerDragon** [comment]
I have to admit that putting just a few spikes in the whole map was intentional. But I totally see your point and I would love to agree on something.

> An alternative would be to make helping blocked tees harder in police.

sounds like a promising path

**problems with killtiles**

- The reason I avoided kill tiles is that I personally enjoy frozen tees and people begging for their help. I enjoy the dominance when a raider looks down on a pool of tees. I enjoy the hope of the frozen tees that keeps them alive.

- BlmapChill provides the option to spend time playing the map and collecting jumps/hook/jetpack etc and this can depend on luck skill and teammates but it usually some effort. It is sad if a player loses that effort quickly or over night while being afk (which is already possible when dragged up to the endless room spikes from police).

- Another big issue with kill tiles in general is that they send you back to spawn. The spawn of BlmapChill is very bad. Both of them are not really spawns of a block map. Multiple tees there are just annoying and stop the flow. I also played a lot of V5 back in the day. I remember spawn being so crowded that navigation there was no fun. If afk a night on v5 you always wake up at spawn which is very demotivating.

**alternative deep**

Deep freeze can solve most if not all my issues with spikes. I hope it can also make helping harder. I have this deep idea for a long time in my head but I did not really find a good spot for the deeps and the undeeps. I also thought about making the pit on the left side so deep that it is impossible to help just with one shotgun but that might turn police into a constant annoying help of friend situation. There is no way a funwar can take place in such an environment.
